components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -726,9 +735,27 @@ public static String getUserConsentURL(OAuth2Parameters params, String loggedInU
      * @param oAuthMessage      oAuth Message.
      * @return                  The consent url.
      */
+    @Deprecated
```

Comment:
Let's point to the method to use

Reply:
Resolve by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2020/commits/a4780e126be8e00ac7e209a2553485da861c8cfb